config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_containerclusters.container.cnrm.cloud.google.com.yaml

```diff
@@ -566,6 +566,10 @@ spec:
                   Enable Binary Authorization for this cluster. If enabled, all container
                   images will be validated by Google Binary Authorization.
                 type: boolean
+              enableCiliumClusterwideNetworkPolicy:
+                description: Whether Cilium cluster-wide network policy is enabled
```

Comment:
AGENT: For better user clarity, please consider enhancing the description to mention that this feature requires the `datapathProvider` to be `ADVANCED_DATAPATH` (Dataplane V2).